src/django_components/component_media.py

```diff
@@ -238,6 +239,13 @@ def __post_init__(self) -> None:
                     f"Received non-null value from both '{inlined_attr}' and '{file_attr}' in"
                     f" Component {self.comp_cls.__name__}. Only one of the two must be set."
                 )
+        # Make a copy of the original state, so we can reset it in tests
+        self._original = copy(self)
+
+    # Return ComponentMedia to its original state before the media was resolved
+    def reset(self) -> None:
+        self.__dict__.update(self._original.__dict__)
```

Comment:
When it comes to component media (template / JS / CSS), these are loaded and populated on access (so when user e.g. tries to access `MyComponent.js`.

To ensure consistency (and because it was triggering error otherwise), the loaded media are discarded after each test.

To be able to restore the "initial" state, the initial state is saved as a copy of the `ComponentMedia` object. Through it we can access the initial state unchanged, whenever needed.